### PR TITLE
💄 update colors

### LINKED ISSUE #20: Update navigation bar to be Plastic Patrol teal
Can we please update the nav bar to be plastic patrol teal (2CB5B5 - we should also update any teal in the app to be this specific shade, just for future reference).

On the leaderboard page, the table header should be plastic patrol blue (78CADE). I've added a drop shadow in the mockup that we probably don't need, but could I please get a screenshot to review before we close this issue?

leaderboard page
about page new

### PR DESCRIPTION
**Pull request by munroTom**
Closes #20 

Screenshot 2020-04-04 at 12 12 23

Screenshot 2020-04-05 at 19 00 53

Screenshot 2020-04-04 at 12 38 05

@joe-mcguinness - think this asset needs updating, not sure if this comes under your responsibilities or not? For now we could leave the background here a slightly different shade if getting a new one may take a bit of time
Screenshot 2020-04-04 at 12 38 19

Screenshot 2020-04-04 at 12 38 53

Screenshot 2020-04-04 at 12 39 01

Screenshot 2020-04-04 at 12 40 34

@joe-mcguinness - don't know what your thoughts are on the disabled text here? Very easy to change if you've got any ideas of something better!
Screenshot 2020-04-04 at 13 23 26

Screenshot 2020-04-04 at 13 31 08

### CODE REVIEW

File: src/components/AboutPage.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ class AboutPage extends React.Component {
           <span style={{ display: "block", textAlign: "center" }}>
             <Button
               onClick={reloadPhotos}
-              color="secondary"
+              color="primary"
```
Comment (munroTom) [inline src/components/AboutPage.js:38]: Screenshot 2020-04-04 at 12 06 31

File: src/components/DrawerContainer.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ class DrawerContainer extends Component {
           </List>
         </div>
 
-        <Typography className={classes.stats} color={"secondary"}>
+        <Typography className={classes.stats} color="primary">
```
Comment (munroTom) [inline src/components/DrawerContainer.js:140]: Screenshot 2020-04-04 at 12 07 57

File: src/components/MapPage/Map.js [hunk 3]
```diff
@@ -354,7 +354,7 @@ class Map extends Component {
           <div>
             <Fab
               className={classes.camera}
-              color="secondary"
+              color="primary"
```
Comment (munroTom) [inline src/components/MapPage/Map.js:357]: Screenshot 2020-04-04 at 12 03 58

File: src/components/common/NavDots/components/NavDots.scss [hunk 4]
```diff
@@ -27,7 +28,7 @@ $navDotsHeight: 10px;
 }
 
 .NavDot__inner {
-  background: $secondary;
+  background: $teal;
```
Comment (munroTom) [inline src/components/common/NavDots/components/NavDots.scss:31]: Screenshot 2020-04-04 at 12 05 30

**Comment by joe-mcguinness** [comment]
Unfortunately I don't have access to that 'see it say it snap it' image from the onboading flow. I'll ask lizzie. We should either replace it now, or leave onboarding from this piece of work and pick it up later

File: src/components/TermsDialog.js [hunk 5]
```diff
@@ -46,17 +46,18 @@ class TermsDialog extends React.Component {
           Welcome to Plastic Patrol
         </DialogTitle>
         <DialogContent>
-          <DialogContentText>
-            Please read our
```
Comment (neilfulwiler) [inline src/components/TermsDialog.js:50]: did we mean to get rid of this?

Comment (munroTom) [reply src/components/TermsDialog.js:50]: Nope, good catch!

**Pull request merged**